Cucumber/src/test/java/com/yourcompany/utils/platforms.yml

```diff
@@ -0,0 +1,15 @@
+windows_10:
+  seleniumVersion: '3.4.0'
+  platform: Windows 10
+  browserName: Chrome
+  version: '71.0'
```

Comment:
maybe use "latest" or "latest-1" for browser versions so the tests don't get outdated that quick.